cli/src/config.rs

```diff
@@ -82,10 +83,7 @@ const fn is_bare_char(b: u8) -> bool {
 }
 
 #[derive(Error, Debug)]
-pub enum ConfigEnvError {
-    #[error("Both {0} and {1} exist. Please consolidate your configs in one of them.")]
-    AmbiguousSource(PathBuf, PathBuf),
-}
+pub enum ConfigEnvError {}
```

Comment:
nit: Can you also remove the error type at all? (or leave a TODO comment?)

Reply:
I left the type here so it's easier to add errors in the future if needed

Reply:
Just delete it unless you have a plan. Reintroducing the error type should be easy.